_delphi_utils_python/delphi_utils/validator/datafetcher.py

```diff
@@ -110,8 +109,12 @@ def get_geo_signal_combos(data_source):
 
     Cross references based on combinations reported available by COVIDcast metadata.
     """
+    params = read_params()
+    assert "validation" in params
```

Comment:
this is a weird check...  it follows the pattern from [run.py](https://github.com/cmu-delphi/covidcast-indicators/blob/1dcfb24453e35b69ee0f0a3387eef3edb541650f/_delphi_utils_python/delphi_utils/validator/run.py#L19-L21), but if this line wasnt here and "validation" was NOT in `params`, the next line would raise its own exception that might be slightly more useful? (`KeyError: 'validation'` instead of `AssertionError`)

¯\\\_(ツ)_/¯ it can stay or go, its up to you